spectrocloud/resource_cloud_account_aws.go

```diff
@@ -25,6 +25,12 @@ func resourceCloudAccountAws() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"context": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "project",
```

Comment:
```suggestion
				Default:      "project",
				Description: "The scope of what the AWS account is available to in Palette. Valid options are either `project` or `tenant.` Defaults to `project`." 
```

I may have this wrong so feel free to correct me gentlemen 😄

Reply:
Looks good to me! Thanks for the example.